tests/cupyx_tests/scipy_tests/sparse_tests/test_csc.py

```diff
@@ -915,10 +915,9 @@ def test_min_sparse_axis_0(self):
                                        cupy.array(dm_data.indptr)),
                                       shape=(10, 20))
 
-        da_cupy_values = cupy.asnumpy(cp_matrix.min(axis=0))
-        da_scipy_values = numpy.array(dm_data.min(axis=0).todense().ravel())
-        da_scipy_values = da_scipy_values[0, :]
-        assert numpy.array_equal(da_cupy_values, da_scipy_values)
+        da_cupy_values = cp_matrix.min(axis=0)
+        da_scipy_values = dm_data.min(axis=0)
+        testing.assert_array_equal(da_scipy_values.A, da_scipy_values.A)
```

Comment:
`testing.assert_array_equal` does not support sparse, while `testing.numpy_cupy_array_equal` and `testing.numpy_cupy_allclose` do.

Reply:
I make the tests use `testing.numpy_cupy_arrray_equal`. Would you check them? To test `nonzero` parameter, which SciPy does not have yet https://github.com/scipy/scipy/pull/11899 and will be marked as experimental in https://github.com/cupy/cupy/pull/3583, I supply a bit messy SciPy inputs.